### DISCUSSION TITLE
Add a "regimen" feature.

### DISCUSSION THREAD

**Discussion by zebbyjz** [monkeytypegame/monkeytype#782] [Ideas] [answered]
Add a regimen feature where people can set the routine they would like to follow for a run.

An example:
1x(english 1k)(expert)
3x(thicc)(quote)
2x(english 10k)(30second)

There could be loads of options and the website will automatically take the user to the next test specified in the regimen.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I don't know if this is a feature that's really required because you could just set everything up yourself. It's not that hard to switch everything to certain settings if you use the site frequently and know where everything is (or just use the command line for mostly everything).

**Comment by TheSirC** [comment]
I get the appeal of the answer : "you can already do it manually" _but_ in an app that promotes to train a skill being able to have a routine where you just select the routine and then you just follow your routine that you can tweak and adapt is interesting to ease the hurdle of the practice and reach flow.
